src/packages/cli/src/__tests__/doctor.test.ts

```diff
@@ -1,74 +1,76 @@
-import tempy from 'tempy'
-import path from 'path'
 import { Doctor } from '../Doctor'
-import copy from '@apexearth/copy'
-import assert from 'assert'
-import stripAnsi from 'strip-ansi'
+import { Context } from './__helpers__/context'
 
-describe('doctor', () => {
-  it('doctor should succeed when schema and db match', async () => {
-    const tmpDir = tempy.directory()
-    await copy({
-      from: path.join(__dirname, 'fixtures/example-project/prisma'),
-      to: tmpDir,
-      recursive: true,
-    })
-    const cwd = process.cwd()
-    process.chdir(tmpDir)
+const ctx = Context.new<{
+  mocked: {
+    'console.error': jest.SpyInstance
+  }
+}>()
 
-    const doctor = Doctor.new()
-    const errLog = []
-    const oldConsoleError = console.error
-    console.error = (...args) => {
-      errLog.push(...args)
-    }
-    const result = await doctor.parse([])
-    console.error = oldConsoleError
+beforeEach(() => {
+  ctx.mocked['console.error'] = jest
+    .spyOn(console, 'error')
+    .mockImplementation(() => {})
+})
+
+afterEach(() => {
+  ctx.mocked['console.error'].mockRestore()
+})
+
+it('doctor should succeed when schema and db do match', async () => {
+  ctx.fixture('example-project/prisma')
+  const result = Doctor.new().parse([])
+  await expect(result).resolves.toEqual('Everything in sync 🔄')
+  expect(ctx.mocked['console.error'].mock.calls).toMatchInlineSnapshot(`
+    Array [
+      Array [
+        👩‍⚕️🏥 Prisma Doctor checking the database...,
+      ],
+    ]
+  `)
```

Comment:
I don't understand why `👩‍⚕️🏥 Prisma Doctor checking the database...` is on stderr 🤔 should be stdout? So maybe in the implementation?

Reply:
Yeah good question, not sure why either